nix/static-haskell-nix.nix

```diff
@@ -0,0 +1,9 @@
+# Pin static-haskell-nix version.
+let
+  static-haskell-nix-rev = "ff7715e0e13fb3f615e64a8d8c2e43faa4429b0f";
```

Comment:
This file can be replaced with the `static-haskell-nix` attribute produced by importing `sources.nix`.

Reply:
Yes, using the `sources.json` is a good idea. It is updated.